### PR TITLE
feat: Created Kafka-To-GCS Python Template and added Documentations

### PR DESCRIPTION
**Pull request by anshumanwins**
#553

### CODE REVIEW

File: python/README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@
 * [RedshiftToGCS](/python/dataproc_templates/redshift#redshift-to-gcs)(blogpost [link](https://medium.com/google-cloud/exporting-data-from-redshift-to-gcs-using-gcp-dataproc-serverless-and-pyspark-9ab78de11405))
 * [SnowflakeToGCS](/python/dataproc_templates/snowflake#1-snowflake-to-gcs)(blogpost [link](https://medium.com/@varunikagupta96/exporting-data-from-snowflake-to-gcs-using-pyspark-on-dataproc-serverless-363d3bed551b))
 * [TextToBigQuery](/python/dataproc_templates/gcs#text-to-bigquery)
+* [KafkaToGCS](/python/dataproc_templates/kafka/#kafka-to-gcs)
```
Comment (vanshaj-bhatia) [inline python/README.md:22]: Move KafkaToGCS to readme in the alphabetical order

File: python/dataproc_templates/kafka/kafka_to_gcs.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+            f'--{constants.KAFKA_TOPIC}',
+            dest=constants.KAFKA_TOPIC,
+            required=True,
+            help='Kafka Topic Name'
+        )
+        parser.add_argument(
+            f'--{constants.KAFKA_STARTING_OFFSET}',
+            dest=constants.KAFKA_STARTING_OFFSET,
+            required=True,
+            help='Starting offset value (earliest, latest)'
```
Comment (vanshaj-bhatia) [inline python/dataproc_templates/kafka/kafka_to_gcs.py:53]: It supports providing json string. Please update the help section accordingly.

File: python/dataproc_templates/kafka/kafka_to_gcs.py [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        )
+        parser.add_argument(
+            f'--{constants.KAFKA_GCS_OUTPUT_FORMAT}',
+            dest=constants.KAFKA_GCS_OUTPUT_FORMAT,
+            required=True,
+            help='Ouput format of the data (json , csv, avro)'
```
Comment (vanshaj-bhatia) [inline python/dataproc_templates/kafka/kafka_to_gcs.py:59]: Does it not support parquet?

**Review by vanshaj-bhatia — CHANGES_REQUESTED** [review]

File: python/dataproc_templates/kafka/kafka_to_gcs.py (hunk 3 above)
Comment (anshumanwins) [reply python/dataproc_templates/kafka/kafka_to_gcs.py]: Yes checked it and added to description

File: python/dataproc_templates/kafka/kafka_to_gcs.py (hunk 2 above)
Comment (anshumanwins) [reply python/dataproc_templates/kafka/kafka_to_gcs.py]: Updated

File: python/README.md (hunk 1 above)
Comment (anshumanwins) [reply python/README.md]: Done

**Review by vanshaj-bhatia — APPROVED** [review]
LGTM! Please proceed with the next steps accordingly, post merging.

**Pull request merged**